druid-shell/src/platform/mac/window.rs

```diff
@@ -167,6 +166,16 @@ impl WindowBuilder {
                 NO,
             );
 
+            if let Some(min_size) = self.min_size {
+                let frame = NSWindow::frame(window);
+                let content_height = window.contentRectForFrameRect_(frame).size.height;
+                let window_height = frame.size.height;
+                let title_bar_height = window_height - content_height;
```

Comment:
Is `height` the only thing that ever has extra space with any style? I don't know much about macOS but it feels to me that if the function returns the `width` as well, we should calculate the `width` difference too.

Reply:
I can't think of an instance where `width` would be meaningful, but I'm also not sure it *doesn't* exist, so using it this does seem more likely to be Correct.

Reply:
Yeah, using it doesn't seem too complex. Also it would future proof us against future style changes.

Reply:
Yeah, in the case that apple adds bottom and side padding back to windows, like was common in Aqua, this would be a source of bugs. good catch, i'll add that in.